lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -350,357 +280,372 @@
         }
       }
     }
+  }
+}
 
-    // ====================
+// +Dashboard - Course
+// ==================== 
+.dashboard .my-courses {
 
-    // UI: individual course item
-    .course {
-      @include box-sizing(box);
-      @include transition(all 0.15s linear 0s);
+  // UI: individual course item
+  .course {
+    @include box-sizing(box);
+    @include transition(all 0.15s linear 0s);
+    @include clearfix();
+    @extend %ui-depth2;
+    position: relative;
+    margin: ($baseline/2);
+
+    .details {
       @include clearfix();
-      @extend %ui-depth2;
-      position: relative;
 
-      .cover {
-        @include box-sizing(border-box);
-        @include transition(all 0.15s linear 0s);
-        overflow: hidden;
-        position: relative;
-        @include float(left);
-        height: 100%;
-        max-height: 100%;
-        width: 200px;
-        height: 120px;
-        margin: 0;
-        border-radius: ($baseline/10);
-        border: 1px solid $dashboard-course-cover-border;
-        border-bottom: 4px solid $dashboard-course-cover-border;
-        padding: ($baseline/10);
+      .wrapper-course-image {
 
-        img {
-          width: 100%;
-          min-height: 100%;
+        .cover {
+          @include box-sizing(border-box);
+          @include transition(all 0.15s linear 0s);
+          @include float(left);
+          @include margin-right(flex-gutter());
+          width: flex-grid(3);
+          overflow: hidden;
+          position: relative;
+          max-height: 100%;
+          height: 120px;
```

Comment:
you could switch this to max-height: 120px if you want it to flex with the image.